lib/erl_interface/doc/src/ei_connect.xml

```diff
@@ -1055,13 +1056,27 @@ if (ei_reg_send(&ec, fd, x.buff, x.index) < 0)
 	<v><seecref marker="#erlang_msg"><c>erlang_msg</c></seecref></v>
       </type>
       <desc>
-        <p>Supports calling Erlang functions on remote nodes.
-          <c>ei_rpc_to()</c> sends an RPC request to a remote node
-          and <c>ei_rpc_from()</c> receives the results of such a
-          call. <c>ei_rpc()</c> combines the functionality of these
-          two functions by sending an RPC request and waiting for the results.
-          See also <seemfa marker="kernel:rpc#call/4">
-          <c>rpc:call/4</c></seemfa> in Kernel.</p>
+        <p>
+          Supports calling Erlang functions on remote nodes.
+          <c>ei_rpc_to()</c> sends an RPC request to a remote node and
+          <c>ei_rpc_from()</c> receives the results of such a
+          call. <c>ei_rpc()</c> combines the functionality of these two
+          functions by sending an RPC request and waiting for the
+          results.
+        </p>
+        <p>
+          The <c>ei_xrpc_to()</c> function is equivalent to
+          <c>ei_rpc_to()</c> when its <c>flags</c> parameter is set to
+          <c>0</c>. When the flags parameter of <c>ei_xrpc_to()</c> is
+          set to <c>EI_RPC_FETCH_STDOUT</c>, stdout (standard output)
+          data are forwarded. See the documentation for the flags
+          parameter for more information about the
+          <c>EI_RPC_FETCH_STDOUT</c> flag.
+        </p>
+        <p>
+        <seemfa marker="kernel:rpc#call/4"> <c>rpc:call/4</c></seemfa>
```

Comment:
```suggestion
        <seemfa marker="kernel:rpc#call/4"><c>rpc:call/4</c></seemfa>
```

Not sure if this space is significant, but may as well remove it.

Reply:
Fixed!